homeassistant/components/camera/netatmo.py

```diff
@@ -20,8 +20,10 @@
 
 CONF_HOME = 'home'
 CONF_CAMERAS = 'cameras'
+CONF_VERIFY_SSL = 'verify_ssl'
```

Comment:
Import and use this -> https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/const.py#L149